src/Promitor.Agents.ResourceDiscovery/Graph/ResourceTypes/KustoClusterDiscoveryQuery.cs

```diff
@@ -0,0 +1,22 @@
+﻿using GuardNet;
+using Newtonsoft.Json.Linq;
+using Promitor.Core.Contracts;
+using Promitor.Core.Contracts.ResourceTypes;
+
+namespace Promitor.Agents.ResourceDiscovery.Graph.ResourceTypes
+{
+    public class KustoClusterDiscoveryQuery : ResourceDiscoveryQuery
```

Comment:
This is more of a general remark, but we should rename all `Kusto` references to `DataExplorer` given that is the product name.
```suggestion
    public class DataExplorerClusterDiscoveryQuery : ResourceDiscoveryQuery
```

Reply:
Renamed respectively.